scripts/adapters.dockerfile

```diff
@@ -17,8 +17,22 @@ FROM $image
 
 COPY scripts/setup-helper-functions.sh /
 COPY scripts/setup-adapters.sh /
-RUN mkdir build && ( cd build &&  source /opt/rh/gcc-toolset-12/enable && \
-    bash /setup-adapters.sh ) && rm -rf build && dnf remove -y conda && dnf clean all
+COPY scripts/setup-centos9.sh /
+RUN mkdir build && \
+    ( \
+      cd build && \
+      source /opt/rh/gcc-toolset-12/enable && \
+      bash /setup-adapters.sh && \
+      bash /setup-centos9.sh install_cuda 12.8 \
```

Comment:
It seems that the version isn't picked up by the function? 1971.9 Error: Unable to find a match: cuda-compat- cuda-driver-devel- cuda-minimal-build- cuda-nvrtc-devel-

Reply:
I fixed this to call `install_cuda` like the CI workflow does: https://github.com/facebookincubator/velox/blob/c2eb47b599c42e80edbb2d9d339ebecd4edbcc09/.github/workflows/linux-build-base.yml#L58-L59